doc/rados/operations/health-checks.rst

```diff
@@ -801,6 +801,92 @@ Or, to disable this alert on a specific OSD, run the following command:
 
    ceph config set osd.123 bluestore_warn_on_spurious_read_errors false
 
+BLOCK_DEVICE_STALLED_READ_ALERT
+_______________________________
+
+There are some logs found in OSD logging triggerred by BlueStore which can 
+cause bad disk. This can lead to degradation of performance in the cluster 
+before identifying the bad disk as the cause.
```

Comment:
This wording is unclear.  How can logs cause "bad disk"?  I think something like the below is in order.

>There are certain BlueStore log messages that surface storage drive issues that can lead to performance degradation.